OneSchedule.Helpers/CollectionNameReader.cs

```diff
@@ -5,7 +5,7 @@ namespace OneSchedule.Helpers
 {
     public static class CollectionNameReader
     {
-        public static string GetName<T>()
+        public static string GetCollection<T>()
```

Comment:
It is better to name it GetCollectionName, because it doesn't retrieve any collection